File: Assets/ShootingRPGKit/CsharpExample/ScriptCS/DatabaseScript/SkillData.cs
using UnityEngine;
using System.Collections;
using UnityEngine.UI;

public class SkillData : MonoBehaviour {
	public Skil[] skill = new Skil[3];
}

[System.Serializable]
public class Skil{
	public string skillName = "";
	public Texture2D icon;
	public Sprite iconSprite;
	public string description = "";
	public Transform skillPrefab;
	public AnimationClip skillAnimation;
	public int manaCost = 10;
	public float castTime = 0.3f;
	public float skillDelay = 0.3f;
}

File: Assets/ShootingRPGKit/Scripts/DatabaseScript/SkillData.js
#pragma strict

class Skil {
	var skillName : String = "";
	var icon : Texture2D;
	var description : String = "";
	var skillPrefab : Transform;
	var skillAnimation : AnimationClip;
	var manaCost : int = 10;
	var castTime : float = 0.3;
	var skillDelay : float = 0.3;
}

var skill : Skil[] = new Skil[3];